src/pocketmine/resources/genisys_eng.yml

```diff
@@ -2,37 +2,37 @@
 
 #Version of this file
 config:
- version: 21
+ version: 22
```

Comment:
Thank you for allowing that, I thought I had to change the version #, I made it easier for it to understand. I will change it! :)